config/routes.rb

```diff
@@ -5,8 +5,10 @@
   resources :topics do
     resources :posts, except: [:index]
   end
-  
-  get "about" => "welcome#about"
+
+  resources :posts do 
```

Comment:
@samehta `routes.rb` looks might be setting up multiple routes for getting to the `PostsController` actions. Reply with the output from running this:

```bash
# Output all routes that route to the PostsController:
rake routes | grep 'posts#'
```